### DISCUSSION TITLE
Hyperformula unable to use

### DISCUSSION THREAD

**Discussion by BharathCL** [handsontable/hyperformula#1385] [General]
Hi,

I'm unable to use hyperformula for 30 days free trail. Please help me.

**Comment by AMBudnik** [comment]
Hi @BharathCL could you please contact us at sales@handsontable.com? We would need to confirm some details.

**Comment by BharathCL** [comment]
Hi Team,

I'm unable to use the Hyperformula in a 30 days free trial build, can you
please do the needful.

Regards,
Bharath.
